.github/workflows/build_cli.yml

```diff
@@ -0,0 +1,50 @@
+name: Ontime CLI build
+
+on:
+  push:
+    tags: [ "*" ]
+  workflow_dispatch:
+
+jobs:
+  build_cli:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 18.18.2
+
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v3
+        with:
+          version: 8
+
+      - name: Install dependencies
+        run: pnpm install --frozen-lockfile
+
+      - name: Build project packages
+        env:
+          SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+        run: pnpm build
```

Comment:
question: what do you think about adding an environment variable here so that we know that the app is running from a CLI build

Reply:
What would be consuming the env variable?

Reply:
We would use it in our setup file to produce a isCLI constant. That would be used by services that benefit from knowing this. For now, the logger

Reply:
I think you would want something that isn't override-able by the person running ontime.

Reply:
If we can find another way that is as convenient and safer it's all good. I don't have strong opinion don't he details of how this should work 

However, I would not rate the fact that users can override it as an important requirement

Reply:
I just meant that the const's like isDocker/isProduction/etc. are not baked in and I could start up the electron app in a way that it thinks it "is Docker" just by setting an env variable on my system. Using an env variable means we would have to get the user to run ontime with that env set. I'll try and see if there is something easy I can do to know if the server is running headless or not.

Reply:
I think I will again leave this to the console logger phase. Still not sure of a good way for the headless version to know it is headless. Maybe a optional argument to `startServer` or something.